### PR TITLE
[Backport] Show tooltips over the cell while typing a function in a cell.

### PR DESCRIPTION
**Pull request by pedropintosilva**
backport of https://github.com/CollaboraOnline/online/pull/8310

-------

commit 1c1b23ea87924669294f4050405ffe2a082f1286
Author: Gülşah Köse <gulsah.kose@collabora.com>
Date:   Mon Feb 19 16:10:06 2024 +0300

    Show tooltips over the cell while typing a function in a cell.
    
    Signed-off-by: Gülşah Köse <gulsah.kose@collabora.com>
    Change-Id: I85354a6c17947824408ae0458f29b858f31ab0e3

### CODE REVIEW

**Comment by pedropintosilva** [comment]
Tested, it works

**Review by GulsahKose — APPROVED** [review]
Works as expected

File: browser/src/layer/tile/CanvasTileLayer.js [hunk 1]
```diff
@@ -2520,6 +2541,39 @@ L.CanvasTileLayer = L.Layer.extend({
...
+		for (var i = 0; i < maxSuggestion; i++) {
+			if (i == 0)
+				resultText = resultText + '[' + functionNameList[i] + ']';
+			else
+				resultText = resultText + ', ' + functionNameList[i];
+		}
+
+		var remainingFuncCount = suggestionArray.length - maxSuggestion;
+		if (remainingFuncCount > 0)
+			resultText = resultText + ' and ' + remainingFuncCount + ' more';
```
Comment (eszkadev) [inline browser/src/layer/tile/CanvasTileLayer.js:2570]: now I noticed not-translated strings here...
It should be:
`resultText + ' ' + _('and %COUNT more').replace('%COUNT, remainingFuncCount)`

(no spaces at the ends of translated string)

Comment (pedropintosilva) [reply browser/src/layer/tile/CanvasTileLayer.js:2570]: good catch!

Comment (GulsahKose) [reply browser/src/layer/tile/CanvasTileLayer.js]: Done thanks

**Review by eszkadev — APPROVED** [review]

**Pull request merged**